fontique/src/backend/fontconfig/config.rs

```diff
@@ -30,6 +30,7 @@ pub fn parse_config(path: &Path, sink: &mut impl ParserSink) {
     if root.tag_name().name() != "fontconfig" {
         return;
     }
+    let mut cachedir_count = 0;
```

Comment:
I'd *probably* just use a boolean here, since the actual count doesn't matter.

Reply:
Updated